### PR TITLE
Fixes for BWV1013

### PR DESCRIPTION
**Pull request by tsiivola**
There is a flat sign missing from a single note in the sixth bar. 

I compared it against [a scan of a manuscript from IMSLP](http://imslp.org/wiki/Partita_in_A_minor%2C_BWV_1013_(Bach%2C_Johann_Sebastian)), specifically the manuscript from the Staatsbibliothek zu Berlin from the year 1722, which is the first entry on the IMSLP page.

### CODE REVIEW

**Comment by tsiivola** [comment]
The repeats were also different than in the manuscript so I changed those as well. Now only the first section is repeated.

File: ftp/BachJS/BWV1013/bwv1013/bwv1013.ly [hunk 1]
```diff
@@ -23,7 +23,7 @@ allemande =  {
     a'16 c''16 e''16 f''16 gis'16 f''16 e''16 d''16 c''16 e''16 gis''16 a''16 e'16 d''16 c''16 b'16 |
     a'16 c''16 gis'16 a'16 f'16 d''16 gis'16 a'16 e'16 c''16 gis'16 a'16 d'16 c''16 b'16 a'16 |
     gis'16 e'16 gis'16 b'16 e''16 gis''16 b''16 d'''16 f''16 d'''16 b''16 gis''16 e''16 d''16 c''16 b'16 |
-    c''16 a'16 c''16 e''16 a''16 c'''16 e'''16 f''16 g''16 b''16 g''16 e''16 c''16 bes'16 a'16 g'16 |
+    c''16 a'16 c''16 e''16 a''16 c'''16 e'''16 f''16 g''16 bes''16 g''16 e''16 c''16 bes'16 a'16 g'16 |
```
Comment (glenl) [inline ftp/BachJS/BWV1013/bwv1013/bwv1013.ly]: +1

**Comment by dominicus** [comment]
Confirmed.
The repeats are better aligned with PD source.

Please update this PR to include additional changes (update to 2.18.2, adds margins, clarifies IMSLP source):

[Link to personal github branch for BWV 1013](https://github.com/dominicus/MutopiaProject/tree/bwv1013/ftp/BachJS/BWV1013)

Or the individual files:
[BWV1013](https://gist.githubusercontent.com/dominicus/7e7fd73f809b455884abd2218149f592/raw/999b993a30768129e6b5c06ea69e583d50a7eaf2/bwv1013.ly)
[BWV1013-sax](https://gist.githubusercontent.com/dominicus/ad3158a770137cf567590b32994117ab/raw/6e579c7189884ec356e336e99532eff9d87134f8/bwv1013_sax.ly)

**Comment by tsiivola** [comment]
Pull request updated with the additional changes.

**Pull request merged**